NetworkPackageDemo/NetworkPackageDemo/Sources/NetworkPackage/RequestBuilder.swift

```diff
@@ -43,6 +43,13 @@ extension RequestBuilder {
         myURL?.queryItems = items
         var request = URLRequest(url: myURL!.url!, cachePolicy: URLRequest.CachePolicy.reloadIgnoringCacheData, timeoutInterval: APIConstants.timeout)
         request.httpMethod = method.rawValue
+        switch method {
```

Comment:
switch method {
        case .get:
           myURL?.queryItems = items
        var request = URLRequest(url: myURL!.url!, cachePolicy: URLRequest.CachePolicy.reloadIgnoringCacheData, timeoutInterval: APIConstants.timeout)
        request.httpMethod = method.rawValue

        default:
/// here use 
            let jsonData = try? JSONSerialization.data(withJSONObject: `parameters`)
            request.httpBody = jsonData
        }
        request.allHTTPHeaderFields = request.allHTTPHeaderFields?.merging(headers, uniquingKeysWith: { (_, newK) -> String in
           
        })

Reply:
Done